public/APIDemo.php

```diff
@@ -4,7 +4,7 @@
 require_once __DIR__ . '/../lib/bootstrap.php';
 require_once __DIR__ . "/../src/RetroAchievementsWebApiClient.php";
 
-RA_ReadCookieCredentials($user, $points, $truePoints, $unreadMessageCount, $permissions);
+RA_ValidateCookie($user, $permissions, $userDetails);
```

Comment:
Instead of adding another optional parameter to `RA_ReadCookieCredentials` just to pass data along to `RenderTitleBar`, `points`, `truePoints`, and `unreadMessageCount` were replaced with a `userDetails` object that is passed to `RenderHeader`, which renders both the titlebar and menu (there's no where that was only doing one or the other).

Most of the rest of the changes in the PR are just applying those changes to every file.